python/generate/templates/configuration.mustache

```diff
@@ -0,0 +1,629 @@
+{{>partial_header}}
```

Comment:
I am assuming that the changes you have introduced here are reflected in the SDK merge request>?

Reply:
Correct, this has the exact same changes. Let me know which order we typically merge these in. Do I merge the lusid-sdk-python-preview changes first, and then these changes? Or vice versa

Reply:
We will probably only commit the SDK changes without the new generated files and then make the generator change afterwards.